static/app/components/compactSelect/index.spec.tsx

```diff
@@ -45,111 +45,277 @@ describe('CompactSelect', function () {
     expect(screen.getByText('Menu title')).toBeInTheDocument();
   });
 
-  it('updates trigger label on selection', function () {
-    const mock = jest.fn();
-    render(
-      <CompactSelect
-        options={[
-          {value: 'opt_one', label: 'Option One'},
-          {value: 'opt_two', label: 'Option Two'},
-        ]}
-        onChange={mock}
-      />
-    );
+  describe('ListBox', function () {
+    it('updates trigger label on selection', function () {
+      const mock = jest.fn();
+      render(
+        <CompactSelect
+          options={[
+            {value: 'opt_one', label: 'Option One'},
+            {value: 'opt_two', label: 'Option Two'},
+          ]}
+          onChange={mock}
+        />
+      );
```

Comment:
Ignore all the indents below. I'm just duplicating these tests, adding another set for grid lists

**from**
```js
it('testA',…)
it('testB',…)
…
```

**to**
```js
describe('ListBox', function() {
  it('testA',…)
  it('testB',…)
  …
})
describe('ListBox', function() {
  it('testA',…)
  it('testB',…)
  …
})
```